src/components/Card/card.css

```diff
@@ -0,0 +1,49 @@
+
+@media screen and (max-width: 600px) {
+  /* Mobile View */
+  .card {
+    max-width: 80%;
+    min-width: 80%;
```

Comment:
setting the max-width and min-width to 90% looks better in my opinion because its the same width as the search bar.